notion-distcopy/src/test/scala/com/ambiata/notion/distcopy/DistCopySpec.scala

```diff
@@ -28,7 +28,7 @@ class DistCopySpec extends AwsScalaCheckSpec(tests = 5) { def is = s2"""
    the full path of the file to copy $createDownloadMappingForADirectory
 
   An upload mapping must contain
-   the full path of the file to copy $createUploadMappingForADirectory
+    the path of the file to copy relative to the from dir $createUploadMappingForADirectory
```

Comment:
Ok, still not a 100% clear to me. But I figured there was a reason (and hence the issue). I think the example would really help clear it up.